packages/blade/src/components/ListView/_decisions/decisions.md

```diff
@@ -0,0 +1,508 @@
+    </ActionList>
+</Dropdown>
+```
+
+### FilterChipGroup
+
+<img src="image-3.png" width="600px" />
+
+#### Usage
+
+```jsx
+const [duration, setDuration] = React.useState();
+const [utrNumber, setUtrNumber] = React.useState();
+const utrNumberInputValueRef = React.useRef(null);
+
+<FilterChipGroup>
+  <FilterChipDatePicker
+    label="Duration"
+    selectionType="range"
+    onChange={(date) => setDuration(date)}
+  />
+
+  {/* FilterChip as trigger for Menu will be released in future and not as part of initial development */}
+  <Menu>
+    <FilterChip value={utrNumber} onClearButtonClick={({ value }) => setUtrNumber(undefined)}>
+      UTR Number
+    </FilterChip>
+    <MenuOverlay>
+      <TextInput ref={utrNumberInputValueRef} />
+      <Button onClick={() => setUtrNumber(utrNumberInputValueRef.current.value)}>Submit</Button>
+    </MenuOverlay>
+  </Menu>
+
+  <Dropdown>
+    <FilterChipSelectInput
+      value={status}
+      defaultValue="Initiated"
+      onClearButtonClick={({ value }) => setStatus(undefined)}
+      label="Status"
+    />
+    <DropdownOverlay>
+      <ActionList>
+        <ActionListItem title="All" onClick={({ name, value }) => setStatus(name)} />
+        <ActionListItem title="Pending" onClick={({ name, value }) => setStatus(name)} />
+        <ActionListItem title="Completed" onClick={({ name, value }) => setStatus(name)} />
+      </ActionList>
+    </DropdownOverlay>
+  </Dropdown>
+</FilterChipGroup>;
+```
+
+#### Props
+
+##### FilterChipGroup
+
+```ts
+type FilterChipGroupProps = {
+  /**
+   * Children prop. Supports Dropdown, Menu components
+   *
+   */
+  children: React.ReactNode;
+};
+```
+
+##### FilterChip
+
+FilterChip can be used as a trigger for Dropdown and Menu
+
+```ts
+type FilterChipProps = {
+  /**
+   * Controlled value of FilterChip.
+   *
+   * FilterChip is always a controlled component since selected state of it comes from other components like Menu, Dropdown.
+   */
+  value: string;
+
+  /**
+   * Change handler when FilterChip's value is cleared
+   */
+  onClearButtonClick: ({ value }: { value: string }) => void;
+
+  /**
+   * Children. Title of the Chip
+   */
+  children: string;
+
+  /**
+   * Disabled state for Chip
+   */
+  isDisabled?: boolean;
+};
+```
+
+### ListView (Layout Component)
+
+Layout Component for handling the overall layout of ListView and Filters. More about layout components at [pattern-terminology.md](./pattern-terminology.md)
+
+```jsx
+<ListView>
+  <ListViewFilters
+    quickFilters={
+      <QuickFilterGroup onChange={({ values }) => console.log(values)}>
+        <QuickFilter title="Status: Completed" value="completed" />
+        <QuickFilter title="Status: Pending" value="pending" />
+      </QuickFilterGroup>
+    }
+    onSearch={({ value, searchType }) => {
+      /* filter on search */
+    }}
+  >
+    <FilterChipGroup></FilterChipGroup>
+  </ListViewFilters>
+  <Table />
+</ListView>
+```
+
+#### Props
+
+##### ListView
```

Comment:
Weren't we going to make the search optional? Can't see any prop to hide search

Reply:
Not adding `onSearch` would remove the search. I'll add a note. Don't think so we need another prop right? 🤔